src/types/interfaces.ts

```diff
@@ -115,6 +115,25 @@ export interface ISdkConfig {
    */
   logger?: ILogger;
 
+  /**
+   * This is name of the app that is consuming the SDK. This field is optional and only
+   *  used for loggering purposes.
+   */
+  originAppName?: string;
+
+  /**
+   * This is the version of the app that is consuming the SDK. This field is optional and only
+   *  used for loggering purposes.
```

Comment:
loggering purposes 😂

Reply:
lol good catch. can you tell I was using my MBP builtin keyboard